resources/hooks/local/pre-push

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+GIT_CURRENT_BRANCH=$(git name-rev --name-only HEAD)
+
+# Fetch the GIT diff and format it as command input:
+DIFF=git diff --name-only origin/"$GIT_CURRENT_BRANCH"..HEAD --oneline
+
+# Run GrumPHP
+(cd "./" && printf "%s\n" "${DIFF}" | exec 'vendor/phpro/grumphp/bin/grumphp' 'git:pre-push')
+
+# Validate exit code of above command
```

Comment:
The script can end with the grumphp command. There is no need for the $RC check.